os/drivers/watchdog.c

```diff
@@ -274,6 +274,9 @@ static int wdog_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	DEBUGASSERT(upper && lower);
 
 	/* Get exclusive access to the device structures */
+	if (cmd < 0) {
+		return -EINVAL;
+	}
 
 	ret = sem_wait(&upper->exclsem);
 	if (ret < 0) {
```

Comment:
I think we'd better return this failure case with errno, not ERROR.

Reply:
you right, i modify that.